File: 03 Berechnungen/Go/Aufgabe5.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strings"
)

var (
	strInput string
	strKlein string
	strGross string
)

func main() {
	fmt.Print("Bitte gib eine Zeichenkette ein: ")

	strInput, _ = bufio.NewReader(os.Stdin).ReadString('\n')

	strKlein = strings.ToLower(strInput)
	strGross = strings.ToUpper(strInput)

	fmt.Println("Deine Zeichenkette:", strInput)
	fmt.Println("Deine Zeichenkette in klein:", strKlein)
	fmt.Println("Deine Zeichenkette in GROSS:", strGross)
}

File: 03 Berechnungen/Pascal/Aufgabe5.pas
Program Aufgabe5;

{$codepage utf8}

Uses sysutils;

Var StrInput, StrKlein, StrGross: String;

Begin
    Write('Bitte gib eine Zeichenkette ein: ');
    Read(StrInput);

    StrKlein := LowerCase(StrInput);
    StrGross := UpperCase(StrInput);

    WriteLn('Deine Zeichenkette: ', StrInput);
    WriteLn('Deine Zeichenkette in klein: ', StrKlein);
    WriteLn('Deine Zeichenkette in GROSS: ', StrGross);
End.